.github/workflows/release.yml

```diff
@@ -74,18 +71,16 @@ jobs:
 
         # back to the release branch
       - uses: actions/checkout@v3
+        with:
+          # tags are needed for the generate-release-contributors.sh script
+          fetch-depth: 0
 
       - name: Generate release notes
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           # conditional blocks not indented because of the heredoc
-          if [[ $VERSION == *.0 ]]; then
-          cat > /tmp/release-notes.txt << EOF
-          This release targets the OpenTelemetry SDK $VERSION.
-
-          EOF
-          else
+          if [[ $VERSION != *.0 ]]; then
```

Comment:
this note:

> This release targets the OpenTelemetry SDK $VERSION.

seems to have snuck in from the instrumentation/contrib repos a while ago, you may want to (manually) remove that note from the release notes for the past few releases